UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/RootBodyPartContainer.cs

```diff
@@ -67,6 +67,14 @@ public class RootBodyPartContainer : MonoBehaviour, IBodyPartDropDownOrgans, ISe
 		[Tooltip("Individual limbs contained within this")]
 		public List<BodyPart> ContainsLimbs = new List<BodyPart>();
 
+		[HideInInspector] public bool IsBleeding = false;
+		
+		/// <summary>
+		/// How much blood does the body lose when there is lost limbs in this container?
+		/// </summary>
+		[Tooltip("How much blood does the body lose when there is lost limbs in this container?")]
+		public float LimbLossBleedingValue = 35f;
```

Comment:
why public?

Reply:
This was public for a reason. But since it's no longer being accessed due to a change requested by you and Bod it's no longer needed to be public. Fixed.

Reply:
sure. Mind the access level of all code you write/modify so the intetion on how to use that code is crystal clear. Give a look to the rest of your code and qustion if they actually need to be public, being it fields or methods.